torch/_inductor/codegen/triton.py

```diff
@@ -72,12 +72,18 @@ def config_of(args):
 
     def is_aligned(x):
         if isinstance(x, TensorArg):
+            known_static_buffer = V.graph.is_fully_static_sized_buffer(x.buffer)
```

Comment:
this test checks if pointer addr is divisible by 16, not if the sizes are divisible by 16. I think it's safe to return "True" here in the same cases as with static.

Reply:
Oh, I see, this is even easier then, we only need sizearg.

I was worried that sizes changing my churn the address, since we no longer specialize, and can have a wildly different tensor.

Reply:
Except for `unaligned_buffers` (I don't know off the top of my head where they are coming from) inductor generates code that passes the base address of the tensor, even if the kernel operates on some slice. Base addresses coming from the allocator are guaranteed to be aligned. 
E.g. if the kernel operates on slice `x[1:]` which will be unaligned, the kernel would look like
```
def triton(x, ...):
    tl.load(x+1, ...)
```
and it will be up to triton to see that `+1` disables vectorized load, but kernel annotation can still say that `x` address is aligned.